pages.tr/linux/a2disconf.md

```diff
@@ -0,0 +1,12 @@
+# a2disconf
+
+> Debian tabanlı işletim sistemlerinde Apache konfigürasyon dosyasını devre dışı bırak.
+> Daha fazla bilgi için: <https://manpages.debian.org/latest/apache2/a2disconf.8.en.html>.
```

Comment:
```suggestion
> Daha fazla bilgi: <https://manpages.debian.org/latest/apache2/a2disconf.8.en.html>.
```
Is icin necessary? If not can you remove it from other files too? [Ref](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py)

Reply:
Its not very necessary but better. In fact those that lack it should have it.